.github/workflows/builder.yml

```
@@ -15,57 +15,86 @@ on:
     - go.mod
 
 env:
-  BUILD_NAME: observer # This needs to match the key in the version file
-  BUILD_TYPE: plugin # Can be plugin, core, supervisor or generic
+  BUILD_NAME: observer
+  BUILD_TYPE: plugin
 
 jobs:
-  build:
-    name: Build plugin
+  init:
+    name: Initialize build
     runs-on: ubuntu-latest
+    outputs:
+      architectures: ${{ steps.info.outputs.architectures }}
+      version: ${{ steps.version.outputs.version }}
+      channel: ${{ steps.version.outputs.channel }}
+      publish: ${{ steps.version.outputs.publish }}
     steps:
     - name: Checkout the repository
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
 
+    - name: Get information
+      id: info
+      uses: home-assistant/actions/helpers/info@info-helper
```

```suggestion
      uses: home-assistant/actions/helpers/info@master
```
Needs <https://github.com/home-assistant/actions/pull/12>